core/src/auth/auth.controller.ts

```diff
@@ -39,4 +43,32 @@ export class AuthController {
     res.send();
     return;
   }
+
+  @Get('auth/createDefaultRoles')
+  async defaultRoles() {
+    // Give superuser access to everything
+    (Object.keys(Resource) as Array<keyof typeof Resource>).map(
+      async (resource) => {
+        (Object.keys(ResourceAction) as Array<keyof typeof ResourceAction>).map(
+          async (action) => {
+            this.addPermTryCatch('superuser', resource, action + ':any');
+          },
+        );
+      },
+    );
```

Comment:
We should add this to the seed script rather than having an endpoint for it; see here:
https://github.com/SprocketBot/sprocket/blob/v2/core/src/db/seed.module.ts#L20-L56

Reply:
Fixed, seeding is done in seed module instead of wonky route.